transaction_decoder/transaction_decoder.py

```diff
@@ -0,0 +1,290 @@
+import base64
+import binascii
+from typing import Any, Dict, List, Optional
+
+from multiversx_sdk_core.bech32 import bech32_encode, convertbits
+
+
+class TransactionToDecode:
+    def __init__(self):
+        self.sender: str = ""
+        self.receiver: str = ""
+        self.data: str = ""
+        self.value: str = "0"
+
+
+class TransactionMetadata:
+    def __init__(self):
+        self.sender: str = ""
+        self.receiver: str = ""
+        self.value: int = 0
+        self.function_name: Optional[str] = None
+        self.function_args: Optional[List[str]] = None
+        self.transfers: Optional[List[TransactionMetadataTransfer]] = None
+
+    def to_dict(self) -> Dict[str, Any]:
+        return {
+            "sender": self.sender,
+            "receiver": self.receiver,
+            "value": self.value,
+            "function_name": self.function_name,
+            "function_args": self.function_args,
+            "transfers": [x.to_dict() for x in self.transfers]
+            if self.transfers
+            else None,
```

Comment:
Can we improve readability a bit here? Or does the auto-formatter break the lines this way?

Reply:
It is the formatter that breaks the lines like this.